src/request.rs

```diff
@@ -166,16 +166,16 @@ mod tests {
 
     #[test]
     fn serde() {
-        use reqwest::header::{ContentLength, UserAgent};
+        use reqwest::header::{CONTENT_LENGTH, USER_AGENT};
 
-        let mut headers = Headers::new();
-        headers.set(ContentLength(2000));
-        headers.set(UserAgent::new("Testing Code"));
+        let mut headers = HeaderMap::new();
+        headers.insert(CONTENT_LENGTH, "2000".parse().unwrap());
```

Comment:
Nitpick: While for strings we really have to `.parse().unwrap()` with the new reqwest version, for numbers we can simply call `.into()` since.

Reply:
Ah, didn't know that one :)